### PR TITLE
feat: release process template

### PR DESCRIPTION
**Pull request by chrispdriscoll**
Create a template we can use for a repeatable release process

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Release Process.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+Internal
+
+## Timing
+After Story 3 (external)
+
+## Action
+https://pages.git.target.com/rapid/doc-site/13-vela/upgrades/#internal and 
+specifically https://pages.git.target.com/rapid/doc-site/13-vela/upgrades/#dev
+
+https://vela.dev.target.com/vela-canary/all-repo restart a build => results in all the other canary repos kicking off a build near the same time (a good way to watch them all is https://vela.dev.target.com/vela-canary/builds)
```
Comment (plyr4) [inline .github/ISSUE_TEMPLATE/Release Process.md:61]: we should remove the internal tgt references

File: .github/ISSUE_TEMPLATE/Release Process.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+
+
+for rc2+ (if needed):
+equivalent stories 1, 4, 5, 6; review 2, 3, 7 for any pertinent changes
+
+
+
+for external release of v0.x.0:
+this cuts the go-vela external release https://pages.git.target.com/rapid/doc-site/13-vela/upgrades/#external (same link as for rc)
```
Comment (plyr4) [inline .github/ISSUE_TEMPLATE/Release Process.md:101]: we should remove the internal tgt references

File: .github/ISSUE_TEMPLATE/Release Process.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+look for bugs
+add vela-canary modifications/additions to story 7
+
+
+
+## Release Story 6: rc1 bugs
+External
+
+## Background
+keeping 1 story for the sake of this summary exercise; could be 1 where we link all bugs external bugs, or separate stories 1 per bug
```
Comment (plyr4) [inline .github/ISSUE_TEMPLATE/Release Process.md:82]: some inconsistent paragraph capitalization, though not limited to this line

File: .github/ISSUE_TEMPLATE/Release Process.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+
+
+## Release Story 7: add/modify / validate canary tests
```
Comment (plyr4) [inline .github/ISSUE_TEMPLATE/Release Process.md:86]: canary testing is done via the internal installation, so maybe convert this step to 
"Validate Internal Installation" or something like that? a different company could run tests however they do?

File: .github/ISSUE_TEMPLATE/Release Process.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+## Release Story 11: release to prod / run prod vela-canary tests
+Internal 
+
+## Timing
+After Story 10
+
+## Action
+https://pages.git.target.com/rapid/doc-site/13-vela/upgrades/#internal and specifically https://pages.git.target.com/rapid/doc-site/13-vela/upgrades/#prod
+https://vela.prod.target.com/vela-canary/all-repo restart a build => results in all the other canary repos kicking off a build near the same time (a good way to watch them all is https://vela.prod.target.com/vela-canary/builds)
```
Comment (plyr4) [inline .github/ISSUE_TEMPLATE/Release Process.md:153]: internal references

**Review by plyr4 — CHANGES_REQUESTED** [review]

**Comment by chrispdriscoll** [comment]
We will review and update. Likely a full overhaul so we'll just close this.

**Pull request closed without merging**